src/playermat/Playmat.ttslua

```diff
@@ -330,6 +330,9 @@ function doUpkeep(_, clickedByColor, isRightClick)
   elseif forcedLearning then
     printToColor("Drawing 2 cards, discard 1 (Forced Learning)", messageColor)
     drawCardsWithReshuffle(2)
+  elseif activeInvestigatorId == "X5U21" then
```

Comment:
What if this character takes forced learning?

Reply:
Fortunately, it is forbidden for her (no permanents).